### PR TITLE
[FIX] Adding 'lang' tag

### PR DESCRIPTION
**Pull request by mariaeduardacunha**
Adding the 'lang' tag to fix accessibility problems

### CODE REVIEW

File: client/startup/i18n.js [hunk 1]
```diff
@@ -56,8 +56,10 @@ Meteor.startup(() => {
 		if (!language) {
 			return;
 		}
-
+		console.log(language);
```
Comment (ggazzo) [inline client/startup/i18n.js]: remove 
```suggestion
```

File: client/startup/i18n.js [hunk 2]
```diff
@@ -56,8 +56,10 @@ Meteor.startup(() => {
...
 		document.documentElement.classList[isRtl(language) ? 'add' : 'remove']('rtl');
+		document.querySelector('html').lang = language;
```
Comment (ggazzo) [inline client/startup/i18n.js]: only on the html tag? what about the meta lang?

File: client/startup/i18n.js [hunk 3]
```diff
@@ -46,7 +46,7 @@ Meteor.startup(() => {
 		});
 	});
 
-	const applyLanguage = (language = 'en') => {
+	const applyLanguage = (language = 'pt-br') => {
```
Comment (ggazzo) [inline client/startup/i18n.js]:
```suggestion
	const applyLanguage = (language = 'en') => {
```

File: client/startup/i18n.js (hunk 1 above)
Comment (mariaeduardacunha) [reply client/startup/i18n.js]: Done

File: client/startup/i18n.js (hunk 3 above)
Comment (mariaeduardacunha) [reply client/startup/i18n.js]: Done

**Pull request merged**